openweathermap-ios/openweathermap-ios/Weather/Model/WeatherManager.swift

```diff
@@ -0,0 +1,47 @@
+import CoreLocation
+
+protocol WeatherManagerDelegate {
+    func didUpdateWeather(_ weatherManager: WeatherManager, weather: WeatherModel)
+}
+
+struct WeatherManager {
+    let weatherURL = "https://api.openweathermap.org/data/2.5/weather?appid=dc05b8287f28b9113a9f84641e77bff5&units=metric"
```

Comment:
Consider let it private.

Reply:
Is this appid sharable?

Reply:
Now api key is used in the android code.

Reply:
Ok, just consider the first comment so.